docs/source/federated-types/sharing-types.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Sharing types and fields
+sidebar_title: Sharing types and fields
+description: Sharing types and fields across multiple subgraphs
+---
+
+import { CodeColumns } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ The field sharing model has changed significantly in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/#type-and-field-sharing). 
+>
+>For more information about using value types in Federation 1, you can view the [previous version of this article here](/federation/value-types/).
+
+In a federated graph, it's common to want to reuse a GraphQL type in multiple [subgraphs](./subgraphs/). 
+
+For example, suppose you want to define and reuse a generic `Position` type in different subgraphs:
+
+```graphql
+type Position {
+  x: Int!
+  y: Int!
+}
+```
+
+This article describes how to share a [GraphQL Object type's](https://www.apollographql.com/docs/apollo-server/schema/schema/#object-types) fields in a federated graph, allowing multiple subgraphs to **redefine** and **resolve** those shared fields. 
```

Comment:
lowercase object

redefine -> define

I think we can get away without bolding these two terms